github/persistance/remind.ts

```diff
@@ -117,3 +117,16 @@ export async function getUserReminder(read:IRead,User:IUser):Promise<IReminder>{
     const index = reminders.findIndex((reminder)=>reminder.userid === User.id);
     return reminders[index];
 }
+
+export async function removeRepoReminder(read:IRead,persistence:IPersistence,repository:string,User:IUser){
+    const reminders = await getAllReminders(read);
+    const idx = reminders.findIndex((u: IReminder) => u.userid === User.id);
+    const repoindex = reminders[idx].repos.findIndex((repo)=>repo == repository);
+
+    if (idx === -1 || repoindex === -1) {
+        return;
+    }
```

Comment:
if idx is -1 this will still break at line 124

Reply:
I fixed it @samad-yar-khan